src/thunderbots/software/ai/passing/evaluation.cpp

```diff
@@ -76,28 +83,34 @@ double AI::Passing::ratePassShootScore(const Field& field, const Team& enemy_tea
         angleSweepCircles(pass.receiverPoint(), field.enemyGoalpostNeg(),
                           field.enemyGoalpostPos(), obstacles, ROBOT_MAX_RADIUS_METERS);
     Angle open_angle_to_goal = Angle::zero();
+    Point shot_target = field.enemyGoal();
     if (shot_opt)
     {
         open_angle_to_goal = shot_opt->second;
     }
 
+    // Figure out what the maximum open angle of the goal could be from the receiver pos.
+    Angle goal_angle = vertexAngle(field.enemyGoalpostNeg(), pass.receiverPoint(),
```

Comment:
Would recommend calculating orientations to each goalpost and using `Angle::minDiff`, since that's more robust and explicit than passing the points in the right order.

Reply:
Changed `vertexAngle` function to `acuteVertexAngle` (and changed implementation).